install/kubernetes/cilium/values.yaml

```diff
@@ -42,141 +40,121 @@ serviceAccounts:
   ui:
     create: true
     annotations: {}
-
 ## If false, agent will not be installed
 ##
 agent: true
-
 ## agent container name
 ##
 name: cilium
-
 ## agent container image
 ##
 image:
   repository: docker.io/cilium/cilium
   tag: latest
   pullPolicy: Always
-
 affinity:
   podAntiAffinity:
     requiredDuringSchedulingIgnoredDuringExecution:
-    - labelSelector:
-        matchExpressions:
-        - key: k8s-app
-          operator: In
-          values:
-          - cilium
-      topologyKey: kubernetes.io/hostname
-
+      - labelSelector:
+          matchExpressions:
+            - key: k8s-app
+              operator: In
+              values:
+                - cilium
+        topologyKey: kubernetes.io/hostname
 ## agent priorityClassName
 ##
 priorityClassName: ""
-
 ## Additional agent container arguments
 ##
 extraArgs: {}
-
 extraEnv: {}
-
 ## Additional InitContainers to initialize the pod
 ##
 extraInitContainers: []
-
 ## Additional agent hostPath mounts
 ##
 extraHostPathMounts: []
-  # - name: textfile-dir
-  #   mountPath: /srv/txt_collector
-  #   hostPath: /var/lib/agent
-  #   readOnly: true
-  #   mountPropagation: HostToContainer
+# - name: textfile-dir
+#   mountPath: /srv/txt_collector
+#   hostPath: /var/lib/agent
+#   readOnly: true
+#   mountPropagation: HostToContainer
```

Comment:
Theses comments diff are annoying as my understanding is that the commented options should be "under" `extraHostPathMounts` (i.e. not at the same level), which is "obvious" before this patch but not after.

Reply:
Thanks for the feedback, I didn't put much thought into this since the yq version I have automatically makes this kind of change. But it sounds like I should put some more effort into finding out if there's a less intrusive implementation that tolerates whitespace in a better way.